components/fonts/platform/freetype/font.rs

```diff
@@ -79,19 +81,25 @@ impl PlatformFontMethods for PlatformFont {
             None => (Au::zero(), Au::zero()),
         };
 
+        let is_variable_font = FT_HAS_MULTIPLE_MASTERS(face.as_ptr());
+        let synthetic_bold =
+            synthetic_bold.is_some_and(|val| if is_variable_font { false } else { val });
+
         Ok(PlatformFont {
             face: ReentrantMutex::new(face),
             requested_face_size,
             actual_face_size,
             table_provider_data: FreeTypeFaceTableProviderData::Web(font_data.clone()),
             variations: normalized_variations,
+            synthetic_bold,
         })
     }
 
     fn new_from_local_font_identifier(
         font_identifier: LocalFontIdentifier,
         requested_size: Option<Au>,
         variations: &[FontVariation],
+        synthetic_bold: Option<bool>,
```

Comment:
Since all the places that use the `Option<bool>` in this PR treat false the same as None, I think it would be simpler to just pass a book everywhere.